core/src/main/java/org/lflang/util/FlexPRETUtil.java

```diff
@@ -0,0 +1,50 @@
+package org.lflang.util;
```

Comment:
I now have this file, which I created heavily inspired by the ArduinoUtil.java file.

My question is, do you know any good candidates for where I should call my `flashTarget` function? I must admit I struggle to get a grasp of what happens where, hehe

Reply:
Good question. It would have to happen sometime after code generation. Probably somewhere in this branch:
https://github.com/lf-lang/lingua-franca/blob/f840daae1414bc32a267fc09fea111300fe01fce/core/src/main/java/org/lflang/generator/c/CGenerator.java#L515

What happens here should be factored out in a bunch of smaller functions, but that's a different issue...